test/Graphics/Image.cpp

```diff
@@ -5,21 +5,26 @@
 #include <GraphicsUtil.hpp>
 #include <array>
 
-TEST_CASE("[Graphics] sf::Image")
+namespace
 {
-    SUBCASE("Default constructor")
-    {
-        const sf::Image image;
-        CHECK(image.getSize() == sf::Vector2u());
-        CHECK(image.getPixelsPtr() == nullptr);
-    }
 
+template <typename... Args>
+sf::Image createSafe(const Args&... args)
```

Comment:
I think a more expressive and simpler solution is to call `.value()` whenever we must get a valid `Image`. If `nullopt` is returned, an exception gets thrown which translates into a test failure.

Reply:
I'm happy to do this in test drivers and examples.